src/core/plugins/spec/actions.js

```diff
@@ -195,12 +178,7 @@ export const executeRequest = (req) => ({fn, specActions, specSelectors}) => {
   // if url is relative, parseUrl makes it absolute by inferring from `window.location`
   req.contextUrl = parseUrl(specSelectors.url()).toString()
 
-
-  if(op && op.operationId) {
-    req.operationId = op.operationId
-  } else if(op && pathName && method) {
-    req.operationId = fn.opId(op, pathName, method)
-  }
+  req.operationId = fn.opId(op, pathName, method)
```

Comment:
TODO: @shockey need feedback on this. 
I needed for reasons, but you've since fixed this upstream correct?

It pertains to executing requests based on operationId